src/midi/midi_fluidsynth.cpp

```diff
@@ -446,6 +449,74 @@ void MidiHandlerFluidsynth::Render()
 	}
 }
 
+std::string format_sf2_line(const std::string &path, const std::string &name)
```

Comment:
Output of this function is _extrenemly_ user-friendly!

Reply:
It's a shame the implementation couldn't be briefer though…